README.md

```diff
@@ -12,13 +12,13 @@ The documentation is built using Markdown pages, which can be converted into a n
 
 The documentation site uses [Hugo](https://gohugo.io/), and specifically uses features found in the extended version of Hugo, v0.76.0 and later.
 
-To install, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version.
+To install, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version. Most users will be able to simply install using the "Homebrew" package manager.
```

Comment:
"Homebrew" (aka `brew`) is almost always endemic to folks running macOS.  So maybe something like:

```suggestion
To install, follow the [installation instructions for your Operating System](https://gohugo.io/getting-started/installing/), and be sure to get the extended version. Most users will be able to simply install using their native package manager like `brew`, `apt` or `snap`.
```